syntax = "proto3";

enum Operation {
  CREATE=0;
  PUT=1;
  DELETE=2;
}

message order {
  int32 id = 1;
  string product_name = 2;
  int32 product_id = 3;
  string user_name =4;
  int32 user_id = 5;
  int32 quantity = 6;
  float total = 7;
  Operation type=8;
}
